style-guide.md

```diff
@@ -0,0 +1,46 @@
+# Style guide
+This document is intended to synchronise styles across OccamResearchLtd outputs, including both R and Excel models.
+
+## Naming conventions
+Variables names should be snake case:
```

Comment:
This is the recommend R naming convention and also v common across languages and in Excel, so I suggest we stick with it.

Reply:
Let’s stick with it 😊 Sounds great!